src/components/ui/toast/index.ts

```
@@ -0,0 +1,10 @@
+export interface ToastProps {
+    state: 'loading' | 'success' | 'error' | 'idle'
+    onReset: () => void
+    onSave: () => void
+    onDismiss: () => void
+}
+
+export function Toast({ state, onReset, onSave, onDismiss }: ToastProps) {
```

ESLint

[error] 'state' is defined but never used.

(@typescript-eslint/no-unused-vars)